src/style/_form.scss

```diff
@@ -31,7 +31,7 @@
 
   &.validation-error {
     > label, > .title {
-      color: #9F3A38;
+      color: #CF0001;
```

Comment:
I color picked #CF0000 from here https://material.io/guidelines/patterns/errors.html#errors-user-input-errors I think its fine to use it.